doc/classes/EditorExportPlatform.xml

```diff
@@ -18,4 +18,18 @@
 			</description>
 		</method>
 	</methods>
+	<constants>
+		<constant name="EXPORT_MESSAGE_NONE" value="0" enum="ExportMessageType">
+			Invalid message type used as the default value when no type is specified.
+		</constant>
+		<constant name="EXPORT_MESSAGE_INFO" value="1" enum="ExportMessageType">
+			Message type for informational messages that have no effect on the export.
+		</constant>
+		<constant name="EXPORT_MESSAGE_WARNING" value="2" enum="ExportMessageType">
+			Message type for warning messages that should be addressed but still allow to complete the export successfully.
```

Comment:
Not sure about `complete the export successfully`, usually warnings are used when non-essential export step fails (changing icons, or code signing), some output is produced, but not necessarily fully usable.

Reply:
How about removing the word `successfully`?

```suggestion
			Message type for warning messages that should be addressed but still allow to complete the export.
```